app/controllers/access_requests_controller.rb

```diff
@@ -1,6 +1,6 @@
 class AccessRequestsController < ApplicationController
   def index
-    @access_requests = AccessRequest.unapproved.order(request_date_utc: :asc)
+    @access_requests = AccessRequestAPI.all
```

Comment:
Where is the `.unapproved.order(request_date_utc: :asc)` now? Is it in the backend? Something about it doesn't feel right ... as in, I think that we should be requesting that the backend apply an order / filter, instead of it being implicit.

Reply:
It's nowhere currently! We will add it back in, as you say explicitly at this call-site would feel better. Will keep this comment around to track that we don't forget about it.

Reply:
Seems like implementing it in the backend using `filter` and `sort` parameters is quite a lot of effort; it doesn't seem to be worth it for this endpoint which will only be used by the support app. I propose we just apply the rigid filtering in the API for now...